applications/nrf5340_audio/README.rst

```diff
@@ -203,7 +205,7 @@ See the following figure for an overview of the synchronization module.
 Both synchronization methods use the SDU reference timestamps (:c:type:`sdu_ref`) as the reference variable.
 If the device is a gateway that is :ref:`using I2S as audio source <nrf53_audio_app_overview_architecture_i2s>` and the stream is unidirectional (gateway to headsets), :c:type:`sdu_ref` is continuously being extracted from the LE Audio Controller Subsystem for nRF53 on the gateway.
 These :c:type:`sdu_ref` values are then sent to the gateway's synchronization module, and used to do drift compensation.
-This extraction of :c:type:`sdu_ref` happens inside the function in :file:`streamctrl.c` that sends encoded data.
+This extraction of :c:type:`sdu_ref` happens inside the :file:`le_audio_xxx_gateway` files' send function.
```

Comment:
Also, _headset.c file is missing from the doc description.

Reply:
What do you mean by "_headset.c file is missing from the doc description"?

Reply:
I mean that with the old version of the docs, we had all files mentioned. Now, we have le_audio_bis/cis_gateway.c mentioned in the docs and these and _headset.c in the figure, but the documentation is missing the mention of _headset.c.

Reply:
This is fixed by the commit made by greg-fer